lib/BloqadeSchema/src/parse.jl

```diff
@@ -1,4 +1,6 @@
 
+error_or_warn(warn::Bool,msg::String) = (warn ? @warn(msg) : throw(ErrorException(msg)))
```

Comment:
I think we should just use the logging system here, you could use the following two macros for warn or errors in logs.

```julia
@warn
@error
```

Reply:
sounds good.